### ISSUE TITLE
[VsCodeExt] Whe Azure Functions Core Tools is not installed, connecting to a Task Hub results in a meaningless 'No response within 60 seconds' error message.

### ISSUE THREAD

**Issue by scale-tone** [microsoft/DurableFunctionsMonitor#41]
Need to detect this condition and make the error message more meaningful. E.g. 'please install Core Tools so and so'

**Comment by mdddev** [comment]
That is so awesome thank you! I will try this as soon as I am Infront of my PC again and will report back here.

**Comment by bremnes** [comment]
FYI: when I tried to run this tool in vscode for the first time, the error message I got actually had that information. "No response within 60 seconds. Ensure you have the latest Azure Functions Core Tools installed globally."

At least for me, that easily pointed me in the right direction on how to get it up and running.

vscode version: 1.66.2
durable functions monitor extension version: v5.5.0

![image](https://user-images.githubusercontent.com/3669798/173223174-6cc98fce-ef34-4f1d-9f9e-05354e5e6178.png)
